automation/hotswap.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
```

Comment:
Could you add some comments at the top describe what the script is for, what args to pass, etc?

Also maybe some code that prints the usage if the arg isn't specified.

These are just good practices for bash scripts.